spec/koans.js

```diff
@@ -427,27 +437,27 @@ describe('destructuring function parameters. ', () => {
   describe('default values', () => {
     it('for simple values', () => {
       const fn = (id, name) => {
-        //expect(id).toEqual(23);
-        //expect(name).toEqual('Bob');
+        expect(id).toEqual(23);
+        expect(name).toEqual('Bob');
       };
-      fn(23);
+      fn(23, 'Bob');
```

Comment:
You can also add Bob as a default value of the name parameter, it will be used if no name is provided in the function call : 
```js
    it('for simple values', () => {
      const fn = (id, name='Bob') => {
        expect(id).toEqual(23);
        expect(name).toEqual('Bob');
      };
      fn(23);
    });
```